src/positioners.js

```diff
@@ -197,21 +198,20 @@ export default {
 		}, config);
 	},
 
-	rect: function(vm, config) {
+	rect: function(elem, config) {
```

Comment:
'rectangle' element was renamed to 'bar'. Not sure if it makes any difference here.

Reply:
Will rename, for consistency.